portifolio.html

```diff
@@ -61,5 +61,83 @@ <h2>Software Developer</h2>
         <li><img src="image/Icon-Medium.png" alt="M" class="media"></li>
     </ul>
 </div>
+
+<div class="arrow">
+    <img src="image/arrow.svg" alt="down-arrow" />
+    <h3>Work</h3>
+</div>
+<div>
+    <h2 class="work">MY RECENT WORK</h2>
+    <img src="image/Vector-4.png" alt="bar" class="img">
+</div>
+<section class="work-section">
+    <div class="placeholder">
+        <div>
+            <img src="image/placeholder.png" alt="blank-placeholder">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        </div>
+        <div>
+            <ul class="items">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li class="ruby">Ruby</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+        </div>
+    </div>
+    
+    <div class="background-image1">
+        <div>
+            <img src="image/placeholder.png" alt="blank-placeholder">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        </div>
+        <div>
+            <ul class="items">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li class="ruby">Ruby</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+        </div>
+    </div>
```

Comment:
- [ ] Kindly make sure the content of each project is aligned using `flexbox`, this is a requirement for this project

Reply:
**It seems template 4 has no About Section**
![temp](https://user-images.githubusercontent.com/95461386/165777567-92f3781a-82c3-4495-9bcb-18af72ce6a01.png)

Reply:
Yes, you are right